PNA.mdk

```diff
@@ -514,11 +514,8 @@ these statements is true:
 ~ End P4Example
 
 ## Match kinds
-~Begin TBD
-Consider simply referencing the corresponding section of the PSA
-specification for this, unless we want to have something different in
-PNA.
-~End TBD
+PNA supports the `match_kinds` specified in section 4.3 of the
+PSA specification.
```

Comment:
The PSA spec doesn't include "optional" as a match kind, which is added in this PR. Should we put an explanation of it here? At least note its existence?

Reply:
The PSA issue is https://github.com/p4lang/p4-spec/issues/1089

Reply:
And was added to the PSA spec with https://github.com/p4lang/p4-spec/commit/3676a3c218e5efd3b32560e546084589cd6b1c64 .

Reply:
but unfortunately not part of a released version of the PSA specification, but that we can fix by Christmas 2022, by releasing updated PSA and PNA specs together.

Reply:
I will see if I can get the "working draft" HTML/PDF links on the specs page to show the actual latest version of PSA and PNA specs (and all of them, really): https://p4.org/specs/